common/libs/values.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type Big from 'big.js';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
I get worried when re-implement stuff like this (just in case there are subtle bugs).

Might be better to use web3's version: https://github.com/ethereum/wiki/wiki/JavaScript-API#web3tohex